### ISSUE TITLE
[Bug]: Review card position error

### ISSUE THREAD

**Issue by 12qweas** [st3v3nmw/obsidian-spaced-repetition#1036]
### Describe the bug

Hello, when I use the review card function, I find that its interface cannot be fully displayed and cannot be dragged. Is this a problem with my computer or the plug-in itself? My OB version is 1.6.7 and the plug-in version is 1.12.5. I look forward to your reply, thank you very much!
<img width="1080" alt="f54521973f153c771a56aa2f0f887b4" src="https://github.com/user-attachments/assets/b2df92e4-2928-4a22-b711-d089aa6b5bea">

### To Reproduce

when I use the review card function, I find that its interface cannot be fully displayed and cannot be dragged.

### Expected behavior

when I use the review card function, I find that its interface cannot be fully displayed and cannot be dragged.

### OS of your device

Win 11

### Obsidian version

1.6.7

### Plugin version

1.12.5.

**Comment by 12qweas** [comment]
I tried changing the height and length percentages of the card, which allowed the options below to be displayed, but it didn't work very well. I think this may be due to the positioning of the review interface (it was originally positioned at the center of the review interface rectangle, but now it is positioned at the upper left corner of the rectangle), or it may be caused by my theme or other plugins. Thank you very much for your reply!
<img width="1080" alt="47088b15ca9bb7dacc0591906eb3806" src="https://github.com/user-attachments/assets/2a227ad6-19a6-4ac2-acf4-5ab81dbe152a">

**Comment by ronzulu** [comment]
Strange. Is this a new installation, or did you upgrade from a previous version?

**Comment by 12qweas** [comment]
> Strange. Is this a new installation, or did you upgrade from a previous version?

Hello, thank you very much for replying! The above is the latest version installed in a brand new library, in order to verify what is causing the above error. The original library plugin was upgraded step by step, but the problem is the same, I would like to ask if this is the only one in this situation? Is your location normal? Thank you!

**Comment by 12qweas** [comment]
> Strange. Is this a new installation, or did you upgrade from a previous version?

Hi, I have found out the cause of the error. It is related to the color scheme of the Blue Topaz theme by Simplicity (@Cuman). You can use this theme to troubleshoot the error. Thank you very much for your help!
<img width="1080" alt="1" src="https://github.com/user-attachments/assets/b16fc174-aeee-4104-aabf-accb48d9b9e1">
<img width="1080" alt="2" src="https://github.com/user-attachments/assets/9935dd15-e0c8-47e5-aa63-09e258da7ded">

**Comment by ronzulu** [comment]
Great, now it's reproducible

![image](https://github.com/user-attachments/assets/16c8354b-ae2c-44ca-bd91-fd88a8d83406)

**Comment by 12qweas** [comment]
Yes, looking forward to your fix, thank you very much!

> 太好了，现在它是可复制的
> 
> ![image](https://private-user-images.githubusercontent.com/75528127/356550944-16c8354b-ae2c-44ca-bd91-fd88a8d83406.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMxOTgzNDIsIm5iZiI6MTcyMzE5ODA0MiwicGF0aCI6Ii83NTUyODEyNy8zNTY1NTA5NDQtMTZjODM1NGItYWUyYy00NGNhLWJkOTEtZmQ4OGE4ZDgzNDA2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODA5VDEwMDcyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU4YmI3ZDQyMjc3OGU2OTNiYjBkZDhhNjg4N2I3ZThkNTQwYzU1MWUzZmYyNzUzY2FhNmMzMjVkNjI5ZTA5NzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.bA13s7FWHYeaRXDnT_q4EppbOSUH4sHjp3iu6nPfbq0)

**Comment by ronzulu** [comment]
Sorry, but I personally won't be able to fix this. I've tried all the different color palette options, and it is only the one you have chosen that has the issue.

![image](https://github.com/user-attachments/assets/66d02415-6f17-4716-b9cf-4b20cfc884e3)

Feel free to leave this issue open, in case someone else is able to solve it. But given that this is such a specific issue that perhaps only affects you, and given the backlog of issues on this project, I wouldn't be particularly optimistic that this will get attention anytime soon.

I'm sure you've already thought of this, but you probably will need to choose a different colour palette.

**Comment by 12qweas** [comment]
Yes, this involves multiple plugins and cannot be solved in a short time. First of all, thank you very much for spending a few hours to help me solve this personal problem; secondly, the theme I am using now has not been updated for a long time due to obsidian reasons, so this may be a historical problem; finally, if someone has the same problem as me, maybe you can try to adjust the aspect ratio of the view to make it display.
Thank you very much for your answer, thank you very much, and wish you a happy life!

**Comment by 12qweas** [comment]
Yes, this involves multiple plugins and cannot be solved in a short time. First of all, thank you very much for spending a few hours to help me solve this personal problem; secondly, the theme I am using now has not been updated for a long time due to obsidian reasons, so this may be a historical problem; finally, if someone has the same problem as me, maybe you (who have the same problem) can try to adjust the aspect ratio of the view to make it display.
Thank you very much for your answer, thank you very much, and wish you a happy life!
<img width="1080" alt="1" src="https://github.com/user-attachments/assets/08236418-3a72-45dd-8bf7-5af29652e34a">
<img width="1080" alt="2" src="https://github.com/user-attachments/assets/afa3a2a6-f97d-43c6-a5ff-91ad7c1481a1">
If you have the same problem, you can change the aspect ratio to 50% and continue to use it.

> Sorry, but I personally won't be able to fix this. I've tried all the different color palette options, and it is only the one you have chosen that has the issue.
> 
> ![image](https://private-user-images.githubusercontent.com/75528127/356586261-66d02415-6f17-4716-b9cf-4b20cfc884e3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMyMDk3MDcsIm5iZiI6MTcyMzIwOTQwNywicGF0aCI6Ii83NTUyODEyNy8zNTY1ODYyNjEtNjZkMDI0MTUtNmYxNy00NzE2LWI5Y2YtNGIyMGNmYzg4NGUzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODA5VDEzMTY0N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQwODgwMDdkZGI2NjEzOWQwN2RhMTI2ZTcwMDdhMmFmZTFjZmUxMThlYTI4OTgyOWVhMmIzMzM3NThjODgzNGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.2UIGxDO7dgPedfKtKBZDV6gfzEVlxPmvezMX3G7cTjE)
> 
> Feel free to leave this issue open, in case someone else is able to solve it. But given that this is such a specific issue that perhaps only affects you, and given the backlog of issues on this project, I wouldn't be particularly optimistic that this will get attention anytime soon.
> 
> I'm sure you've already thought of this, but you probably will need to choose a different colour palette.
